### PR TITLE
Add architecture diagrams

### PR DESCRIPTION
**Pull request by Silvan-WMDE**
This adds a first version of Mismatch Finder's architecture diagram in
diagrams.net (SVG) format.

### CODE REVIEW

**Review by itamargiv — DISMISSED** [review]
Nice! This drawio is a a super cool extension as well!

**Comment by itamargiv** [comment]
So I've put all my thoughts to paper so far. The only thing I'm not really sure about is this whole Upload mechanism:
1. Should it even be a restful endpoit?
2. Do we really need to store uploads in the FS?

**Comment by Silvan-WMDE** [comment]
I like it - thanks @itamargiv!

File: diagrams/arch_diagram.drawio.svg [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        <g fill="#000000" font-family="Helvetica" text-anchor="middle" font-size="14px">
+            <text x="517.5" y="223.15">
+                Website
```
Comment (mariushoch) [inline diagrams/arch_diagram.drawio.svg:138]: Is this the repo wiki (Wikidata)?

**Review by guergana — APPROVED** [review]
LGTM! :)

**Pull request merged**